step1_pred_pairwise_novgg_withoutcap.py

```diff
@@ -0,0 +1,458 @@
+# coding: utf-8
```

Comment:
Seems that most of the functions here and in _withcap.py are the same. Please refactor the identical functions into a common library (file).
Ideally, cap and no_cap should be a flag given to the same file, but if that's too much work, at least lets refactor the common functions.

Reply:
I've merged two files